tests/unit/frameworks/dendrites/dendrites_test.py

```diff
@@ -0,0 +1,168 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+import unittest
+
+import torch
+
+from nupic.research.frameworks.dendrites import (
+    DendriteSegments,
+    DendriticWeights,
+    GatingDendriticWeights,
+)
+
+
+class DendriteSegmentsTests(unittest.TestCase):
+
+    def test_forward(self):
```

Comment:
We should add a test where the dendrites-as-bias units actually output the expected values. Please see https://github.com/numenta/nupic.torch/blob/master/tests/k_winners_test.py as an example.